charts/sourcegraph/values.yaml

```diff
@@ -144,6 +144,10 @@ codeInsightsDB:
       value: password
   # -- Name of existing ConfigMap for `codeinsights-db`. It must contain a `postgresql.conf` key.
   existingConfig: "" # Name of an existing configmap
+  # -- Additional PostgreSQL configuration. This will overwrite our default configuration.
```

Comment:
This wording is a bit unclear - it makes it sound like the entire configuration will be overwritten. Instead, we should make it obvious that this is to provide overrides or extend the existing configuration - not replacing it wholesale (like the `existingConfig` option).

Reply:
I was probably thinking "override" in my brain but ended up typing "overwrite"